lib/constants.js

```diff
@@ -5,6 +5,8 @@ module.exports = Object.freeze({
   BOSH_POLL_MAX_ATTEMPTS: 3,
   DEPLOYMENT_LOCK_NAME: '_LOCK_',
   SERVICE_FABRIK_PREFIX: 'service-fabrik',
+  NOT_APPLICABLE: 'NA',
+  FINISHED_JOBS_RETENTION_DURATION_DAYS: 14,
```

Comment:
is it better to make configurable the value 14?

Reply:
Thought about this. These are useless one time job definitions which are no longer valid. No real need to change the configuration of these on the fly. Job runs are anyways retained. There would not be any time a pressing need to change them immediately.